src/models/projects.js

```diff
@@ -0,0 +1,96 @@
+const db = require("../database");
+const tb = "projects";
+
+exports.list = id => {
+  if (id) {
+    return db(tb)
+      .where({ id })
+      .first()
+      .then(project => ({
+        ...project,
+        completed: project.completed === true ? true : false
```

Comment:
if you're writing this line as much as you are, I would write a function and put it at the top and invoke that function here....this is also where I would set your default to false....you can do that pretty easily by doing this: 

```
completed:  invokedFunc() || false;
```
if the invoked Func exists, it'll return that answer, if not it'll return false automagically. This will prevent the migrations table from overwriting a truthy completed property when entered with a false.